testing/perl-crypt-openssl-aes/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Automatically generated by apkbuild-cpan, template 1
+# Maintainer: Sergey Safarov <s.safarov@gmail.com>
+pkgname=perl-crypt-openssl-aes
+_realname=Crypt-OpenSSL-AES
+pkgver=0.02
+pkgrel=0
+pkgdesc="Perl module Crypt-OpenSSL-AES"
+url="http://search.cpan.org/~ttar/Crypt-OpenSSL-AES-0.02/"
+arch="all"
+license="Artistic-1.0 GPL+"
+cpandepends=""
```

Comment:
A lot of these variables are not used or required,  I suggest you remove empty variables:

```suggestion
```